controllers/apartments.rb

```diff
@@ -8,7 +9,18 @@
   erb :"apartments/new"
 end
 
+#create
+post "/apartments" do
+  new_apartment = Apartment.create(address: params[:address],
+                                   monthly_rent: params[:monthly_rent],
+                                   sqft: params[:sqft],
+                                   num_beds: params[:num_beds],
+                                   num_baths: params[:num_baths])
```

Comment:
This approach to creating isn't wrong per se, but the later way we used in Tunr is generally preferred. By that, I mean where we 'namespace the form inputs, to look like:
`<input name="apartment[address]>`
instead of just
`<input name="address">`.

This way, we can just do the following in our controller:
`new apartment = Apartment.create(params[:apartment]`